lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,597 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_facts
+short_description: Gathers facts about remote PostgreSQL servers.
```

Comment:
The short description does not need a trailing dot (all other descriptions do).
```suggestion
short_description: Gather facts about remote PostgreSQL servers
```